src/Routes/RosSystemDetail/RosSystemDetail.js

```diff
@@ -51,7 +51,7 @@ class RosSystemDetail extends React.Component {
         const chrome = this.props.chrome;
         const displayName = this.props.rosSystemInfo.display_name;
         if (displayName && displayName !== document.title) {
-            chrome?.updateDocumentTitle(displayName);
+            chrome?.updateDocumentTitle(`${displayName} - Resource Optimization - Business`);
```

Comment:
@mkholjuraev for now can you please put the title in a variable and use it? for eg:
`const detailPageTitle = `${displayName} - Resource Optimization - Business`;
 chrome?.updateDocumentTitle(detailPageTitle);`

I checked the lint is not complaining with this. I will look into the lint issue later around why it gives error with above code. 
Thank you!